docs/static-s3-website.md

```diff
@@ -0,0 +1,78 @@
+## Construct StaticS3Website
+
+This construct creates an S3 bucket and upload any content into it.
```

Comment:
Todo: change to "This construct creates a S3 bucket and uploads the contents of a local folder to it"